bpf/flows.c

```diff
@@ -171,6 +174,48 @@ static inline int flow_monitor(struct __sk_buff *skb, u8 direction) {
             }
         }
     }
+
+    // Update additional metrics (per-CPU map)
+    if (pkt.dns_id != 0 || dns_errno != 0) {
+        // hack on id will be removed with dedup-in-kernel work
+        id.direction = 0;
+        id.if_index = 0;
```

Comment:
will the feature work with this hack ?

Reply:
sure, it's precisely here to make it work :-)
On user-space we assume all additional map entries are without dir/interface. This is temporary, my other PR is removing that, since eventually we want to entirely get rid of interface/direction in flow key.